src/dev/build/tasks/os_packages/docker_generator/resources/bin/kibana-docker

```diff
@@ -136,7 +136,7 @@ kibana_vars=(
     xpack.reporting.queue.timeout
     xpack.reporting.roles.allow
     xpack.searchprofiler.enabled
-    xpack.security.authProviders
+    xpack.security.authc.providers
```

Comment:
See https://github.com/elastic/kibana/pull/37346#discussion_r290622120, I also want to remove `rename(....)` deprecation from master PR (just a left over from local testing) and keep it only in 7.x backport. Does it sound good to you?

Reply:
Sure, that's fine by me.

Reply:
I don't know if it simplifies or complicates things, but to handle situations like this previously, I've made the main PR implement the 7.x behavior and then followed up with another PR after the fact which only be applied to master to make the breaking change.

Since we still have quite a few 7.x releases, and to make backporting easier, I've before held off on making the breaking change to master until much later in the 7.x cycle, if we aren't worried about consumers needing to adjust to the change which we've made.

Reply:
Yeah, thanks for sharing, that makes sense! And it's easier for review to follow as well. I'll do that the way you suggest.